admin/inc/inc.php

```diff
@@ -0,0 +1,26 @@
+<?php
+# *** LICENSE ***
+# This file is part of BlogoText.
+# http://lehollandaisvolant.net/blogotext/
+#
+# 2006      Frederic Nassar.
+# 2010-2016 Timo Van Neerden.
+#
+# BlogoText is free software.
+# You can redistribute it under the terms of the MIT / X11 Licence.
+#
+# *** LICENSE ***
+
+/**
+ * all file in /admin/inc/*.php
+ * must be include here
+ */
+
+
+require_once get_path('/inc/auth.php');
```

Comment:
Voici ce que ça renvoie :

    Warning: require_once(/mnt/master/www/local/blogotext/admin): failed to open stream: No such file or directory in /mnt/master/www/local/blogotext/admin/inc/inc.php on line 17

    Fatal error: require_once(): Failed opening required '' (include_path='.:/usr/share/php:/usr/share/pear:') in /mnt/master/www/local/blogotext/admin/inc/inc.php on line 17

Pour le debug :

    require_once get_path('/inc/auth.php', true, true);
    string(60) "get_path() : seem's already an absolute path (/inc/auth.php)"